lib/supply/uploader.rb

```diff
@@ -1,14 +1,15 @@
 module Supply
   class Uploader
     def perform_upload
-      FastlaneCore::PrintTable.print_values(config: Supply.config, hide_keys: [:issuer], title: "Summary for supply #{Supply::VERSION}")
+      # FIXME hide :json
```

Comment:
Annotation keywords like `FIXME` should be all upper case, followed by a colon, and a space, then a note describing the problem.